assets/src/edit-story/elements/shape/index.js

```diff
@@ -45,6 +45,8 @@ export const resizeRules = {
   vertical: true,
   horizontal: true,
   diagonal: true,
+  minWidth: 20,
```

Comment:
These values aren't real pixels, meaning that on smaller screens, this would look smaller than 20px which might not even be accessible anymore (too small). Just to confirm, is that OK? Perhaps it is, at least it doesn't go to 0 anymore.

Reply:
Yes, this is 20px on canvas (20px on-screen with 100% canvas size). Making it 20px on-screen (always) would mean you can resize it differently on small vs big canvas, which might feel weird.

Reply:
We definitely shouldn't make it always 20px on-screen but let's follow up with UX to confirm the 20 min size considering small screens, too. Figma says 20px but I'm not sure if that considers that the 20px is not real px.

I can follow up on this to leave it out of this PR.